src/Microsoft.AspNet.Mvc.Core/Controllers/FilterActionInvoker.cs

```diff
@@ -178,7 +178,7 @@ private set
             _cursor = new FilterCursor(_filters);
 
             ActionContext.ModelState.MaxAllowedErrors = _maxModelValidationErrors;
-
+            
```

Comment:
It may display as a space on github, but it's a rarely used unicode character that increases your performances. `\u1337`